regulation-worker/internal/delete/api/api_test.go

```diff
@@ -670,6 +776,37 @@ func (cpRespProducer *cpResponseProducer) mockCpRequests() *chi.Mux {
 			return
 		}
 	})
+
+	srvMux.HandleFunc("/workspaces/{workspaceId}/destinations/{destinationId}/authStatus/toggle", func(w http.ResponseWriter, req *http.Request) {
+		if req.Method != http.MethodPut {
+			w.WriteHeader(http.StatusNotFound)
+			return
+		}
+		// iterating over request parameters
+		for _, reqParam := range []string{"workspaceId", "destinationId"} {
+			param := chi.URLParam(req, reqParam)
+			if param == "" {
+				// This case wouldn't occur I guess
+				http.Error(w, fmt.Sprintf("Wrong url being sent: %v", reqParam), http.StatusInternalServerError)
```

Comment:
If the server gets an empty parameter, isn't that a Bad Request rather than a 500?